acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/transaction/ibft2/BftRequestFactory.java

```diff
@@ -28,7 +28,7 @@
 import org.web3j.protocol.core.Request;
 import org.web3j.protocol.core.Response;
 
-public class Ibft2RequestFactory {
+public class BftRequestFactory {
```

Comment:
Think should move this into the a bft package

Reply:
changed whole package to BFT